vue.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  transpileDependencies: ["vuetify"],
+  devServer: {
+    proxy: {
+      "/login": {
+        ws: true,
+        changeOrigin: true,
+        target: "http://localhost:8000"
+      }
```

Comment:
what is it?

Reply:
It is config which allow communication with api and avoid CORS when use localhost

Reply:
You don't need these additional options: `ws` (that's for websockets) and `changeOrigin` (not 100% sure about that one). Besides, you'd probably want to proxy all API request, not just the `/login` path. Maybe use `/api` prefix for that?